odoolib/main.py

```diff
@@ -118,14 +118,15 @@ def json_rpc(url, fct_name, params):
     result = result_req.json()
     if result.get("error", None):
         raise JsonRPCException(result["error"])
+    result["result"] = result.get("result", None)
```

Comment:
Why not simply return result.get("result", None) ?

Reply:
I agree return result.get("result", None) more simple.